static/scss/partials/dashboard.scss

```
@@ -5,6 +5,10 @@
 
 .mpp-dashbaord-header-name {
     font-weight: 700;
+    width: 100%;
+    word-wrap: break-word;
+    padding-right:5%;
```

nit: whitespace

And also – percentage padding feels funky here. Does it work with a token? 

```suggestion
    padding-right: 5%;
```

---

Switched out the percentage for `$spacing-sm` instead! 😄